.nvmrc

```
@@ -1 +1 @@
-16.19
+18.15.0
```

```suggestion
18.15
```
I think we probably want to allow patches, so pinning the major and minor only.
I'm not 100% sure how we work with PaaS yet but I _think_ the latest buildpack available for nodejs may be 18.14.  I might be really wrong about that/it might not matter here

---

🤦‍♂️ Sorry @cadmiumcat  Only just seen your comment after merging it from Dans 👍. 

I've had problems in the past with `asdf`(my local version manager for ruby & node) where it needs to know what minor version we are using. Oddly it doesn't seem to be an issue with forms-runner where the nvmrc pins minor and patch

---

Yeah I was wondering the same, I think Node is necessary for the asset compilation which when deploying to PaaS we seem to do that before cf push https://github.com/alphagov/forms-admin/blob/main/.github/workflows/deploy.yml#L24